lib/declaration.ts

```diff
@@ -692,6 +692,10 @@ export default class DeclarationTranspiler extends base.TranspilerBase {
       case ts.SyntaxKind.ProtectedKeyword:
         // Handled in `visitDeclarationMetadata` below.
         break;
+      case ts.SyntaxKind.DeclareKeyword:
+        // In an ambient top level declaration like "declare var" or "declare function", the declare
+        // keyword is stored as a modifier but we don't need to handle it.
```

Comment:
nit: add a comment explaining why we don't need to handle it.

Reply:
Added, let me know if it works